ci/scripts/execute_build.sh

```diff
@@ -94,7 +92,7 @@ fi
 
 GRADLE_COMMAND="./gradlew \
     ${DEFAULT_GRADLE_TASK_OPTIONS} \
-    build install"
+    build install -x test"
```

Comment:
I wonder if realistically we can just drop this job altogether since all the xTest targets build anyway. The finally artifact that gets published isn't even from this build.